inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerSidebarComponent.tsx

```diff
@@ -54,17 +75,27 @@ export const PerformanceViewerSidebarComponent = (props: IPerformanceViewerSideb
             {metadataCategories && metadataCategories.map((category) => (
                 <div key={`category-${category || 'version'}`}>
                     {category
-                        ? <div className="category-header header" key={`header-${category}`}>
-                            <span>{category}</span>
-                            <input type="checkbox"/>
+                        ? <div className="category-header header sidebar-item" key={`header-${category}`}>
+                            <span className="category">{category}</span>
+                            <CheckBoxLineComponent isSelected={() => metadataCategoryChecked?.get(category) === metadataCategoryId?.get(category)?.length} onSelect={onCheckAllChange(category)} faIcons={{faIconEnabled: faCheckSquare, faIconDisabled: faSquare}} />
                           </div>
-                        : <div className="version-header header" key={"header-version"}>Version:</div>}
+                        : <div className="version-header header sidebar-item" key={"header-version"}>
+                            <span className="category">Version:</span>
+                            <span className="value">100</span>
+                        </div>}
                     {metadataCategoryId?.get(category)?.map((id) => {
                         const metadata = metadataMap?.get(id);
-                        return metadata && <div key={`perf-sidebar-item-${id}`} className="sidebar-item">
-                            <input type="checkbox"checked={!metadata.hidden} onChange={onCheckChange(id)} />
-                            <ColorPickerLineComponent value={Color3.FromHexString(metadata.color ?? "#000")} onColorChanged={onColorChange(id)} shouldPopRight />
-                            <span className="sidebar-item-label">{id}</span>
+                        return metadata && <div key={`perf-sidebar-item-${id}`} className="sidebar-item measure">
+                            {/* div with check box, color picker and category name */}
+                            <div className="category">
+                                <CheckBoxLineComponent isSelected={() => !metadata.hidden} onSelect={onCheckChange(id)} faIcons={{faIconEnabled: faCheckSquare, faIconDisabled: faSquare}} />

```

Comment:
i think there is no need for the `faIcon` in the keys, it is already defined in the name of the parameter.